.github/workflows/test.yml

```diff
@@ -386,6 +386,30 @@ jobs:
         with:
           name: screenshots
           path: tmp/screenshots/
+  react-component-tests:
```

Comment:
Let's have consistent CI stage naming and call this 'test-react-components'

Reply:
done